tensorboard/plugins/graph/tf_graph_loader/tf-graph-loader.ts

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 namespace tf.graph.loader {
   Polymer({
     is: 'tf-graph-loader',
-
+    _template: null,
```

Comment:
Can we add the same comment here, e.g. ` // strictTemplatePolicy requires a template (even a null one).`